src/UserSpaceInstrumentation/InstrumentProcess.cpp

```diff
@@ -365,6 +367,7 @@ ErrorMessageOr<std::unique_ptr<InstrumentedProcess>> InstrumentedProcess::Create
   void* library_handle = library_handle_or_error.value();
 
   // Get function pointers into the injected library.
+  ORBIT_LOG("Resolve function pointers in injected library.");
```

Comment:
```suggestion
  ORBIT_LOG("Resolving function pointers in injected library");
```
We don't usually use periods at the end of log lines (unlike messages shown to the user).